resources/access.rb

```diff
@@ -16,12 +16,12 @@
 # limitations under the License.
 #
 
-property :access_type,   String, required: true, default: 'local'
```

Comment:
Ho come we're getting rid of required here?

Reply:
I got those errors, default and required are mutually exclusive ref: https://app.circleci.com/jobs/github/sous-chefs/postgresql/1226